babylon/assetlist.json

```diff
@@ -1095,6 +1095,59 @@
           }
         }
       ]
+    },
+    {
+      "description": "MilkyWay's liquid staked BABY",
+      "denom_units": [
+        {
+          "denom": "ibc/CD7ECDBBA538632B45915ED67C4932DDB318818C00C21489D3774E560E87E420",
+          "exponent": 0
+        },
+        {
+          "denom": "milkBABY",
+          "exponent": 6
+        }
+      ],
+      "type_asset": "ics20",
+      "base": "ibc/CD7ECDBBA538632B45915ED67C4932DDB318818C00C21489D3774E560E87E420",
+      "name": "milkBABY",
+      "display": "milkBABY",
+      "symbol": "milkBABY",
+      "traces": [
+        {
+          "type": "liquid-stake",
+          "counterparty": {
+            "chain_name": "babylon",
+            "base_denom": "ubbn"
+          },
+          "provider": "MilkyWay"
+        },
+        {
+          "type": "ibc",
+          "counterparty": {
+            "chain_name": "milkyway",
+            "base_denom": "factory/milk1qg5ega6dykkxc307y25pecuufrjkxkaggkkxh7nad0vhyhtuhw3ssgcye4/umilkBBN",
+            "channel_id": "channel-4"
```

Comment:
Validation is still failing because the IBC connection between MilkyWay and Babylon has not yet been registered.

Reply:
Resolved it